lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h

```diff
@@ -262,11 +262,14 @@ DECLARE_MODEL(api_event, ZTX_API_EVENT_MODEL)
 
 typedef struct tunneled_service_s tunneled_service_t;
 
+#define MAX_WRITE_SIZE (128 * 1024)
+
 /** context passed through the tunneler SDK for network i/o */
 typedef struct ziti_io_ctx_s {
     ziti_connection      ziti_conn;
     bool ziti_eof;
     bool tnlr_eof;
+    uint64_t active_writes;
```

Comment:
it looks like this is actually being used to track pending bytes? if so, this name is misleading.

Reply:
ahh, much better name